brayns/common/Transform.h

```diff
@@ -29,7 +28,7 @@ namespace brayns
  * @brief Defines the translation, rotation and scale parameters to be applied
  * to a scene asset.
  */
-class Transform : public BaseObject
+class Transform
```

Comment:
If it is not a BaseObject anymore, maybe we can make a struct with public members to lighten a bit the writing no ? (as you prefer)